provider/cmd/run.go

```diff
@@ -72,6 +72,7 @@ const (
 	FlagOvercommitPercentStorage         = "overcommit-pct-storage"
 	FlagDeploymentBlockedHostnames       = "deployment-blocked-hostnames"
 	FlagAuthPem                          = "auth-pem"
+	FlagKubeConfig                       = "kube-config"
```

Comment:
shall we declare flag as `kubeconfig` instead? this way it will match to env variable `AKASH_KUBECONFIG` (which is pretty close to original `KUBECONFIG`) instead of `AKASH_KUBE_CONFIG`

Reply:
Yeah, I'll make this change.